lib/routes/v1.js

```diff
@@ -102,6 +103,47 @@ function discoverKey(content) {
     return null;
 
 }
+
+/**
+ * @function saveKeyToFile
+ * Write content in a file, using region name for file name
+ * @param {String} region
+ * @param {Object} req
+ * @param {Object} res
+ */
+function saveKeyToFile(region, content, res) {
+        if (region === undefined) {
+            res.status(401);
+            res.end();
+        } else {
+            console.log('post new key for region ' + region);
+            var key = discoverKey(content);
+            if (key === null) {
+                throw new Error('invalid key');
+            }
+
+            var filename = keysPath + region + '.' + key + 'key';
+
+            fs.writeFile(filename, content, function(err) {
+                if (err) {
+                    return console.log(err);
+                }
+
+                console.log('The key file %s was saved!', filename);
+                res.status(201);
+                res.type('text/plain');
+                res.setHeader('Content-Length', content.length);
+                res.write(content, 'utf8');
+                res.end();
+
+            });
+        }
+
+
+
+
```

Comment:
yes, you could see here: https://travis-ci.org/telefonicaid/fiware-aiakos/builds/97618645#L402